java/src/jmri/jmrit/operations/rollingstock/RollingStockEditFrame.java

```diff
@@ -63,7 +63,7 @@ public abstract class RollingStockEditFrame extends OperationsFrame implements j
 
     public JButton saveButton = new JButton(Bundle.getMessage("ButtonSave"));
     public JButton deleteButton = new JButton(Bundle.getMessage("ButtonDelete"));
-    public JButton addButton = new JButton(Bundle.getMessage("ButtonAdd")); // TODO have button state item to add
+    public JButton addButton = new JButton(); // add car or locomotive
```

Comment:
I assume this change is intended. But I noticed that this button now doesn't have any text, unless you give it a text later. I will approve and merge this PR and there is no need to comment on this if it's OK. But I write this comment in case it's a misstake.